src/lib/core.js

```diff
@@ -1643,6 +1643,10 @@ export default function (context, pluginCallButtons, plugins, lang, options, _re
             _w.setTimeout(function () {
                 if (typeof plugin.select === 'function') this.callPlugin(pluginName, plugin.select.bind(this, element), null);
                 this._setComponentLineBreaker(element);
+
+                const compContext = context[pluginName];
+                const container = compContext._container;
+                core.setRange(container, 0, container, 0);

```

Comment:
This code is a problem when entering text after selecting an image.
After the merge, this code will be removed.
Thank you for your contribution.!

Reply:
This change is crucial for my other PR's.
Are you talking about pressing text keys when there is a selected component?
If I get you right, this problem was earlier.
Does it get fixed with next PR: https://github.com/JiHong88/suneditor/pull/1267/commits ?

Reply:
I highly recommend including this code. Just a friendly reminder so you don't forget it when the time to work with PRs will come. @JiHong88